src/content/en/fundamentals/progressive-web-apps/prpl.md

```diff
@@ -0,0 +1,243 @@
+
+-   A bundled build designed to minimize the number of round-trips required to get the application
+	running on server/browser combinations that don't support server push.
+
+Your server logic should deliver the appropriate build for each browser.
+
+### Bundled build
+
+For browsers that don't handle HTTP2 Push, the build process could produce a set
+of different bundles: one bundle for the shell, and one bundle for each
+fragment. The diagram below shows how a simple app would be bundled, again using
+Web Components:
+
+![diagram of the same app as before, where there are 3 bundled
+dependencies](images/app-build-bundles.png)
+
+Any dependency shared by two or more fragments is bundled in with the shell and
+its static dependencies.
+
+Each fragment and its _unshared_ static dependencies are bundled into a single
+bundle. The server should return the appropriate version of the fragment
+(bundled or unbundled), depending on the browser. This means that the shell code
+can lazy-load `detail-view.html` _without having to know whether it is bundled
+or unbundled_. It relies on the server and browser to load the dependencies in
+the most efficient way.
+
+
+## Background: HTTP/2 and HTTP/2 server push
+
+HTTP/2 allows _multiplexed_ downloads over a single connection, so that multiple
+small files can be downloaded more efficiently.
+
+HTTP/2 server push allows the server to preemptively send resources to the
+browser.
+
+For an example of how HTTP/2 server push speeds up downloads, consider how the
+browser retrieves an HTML file with a linked stylesheet.
+
+In HTTP/1:
+
+*   The browser requests the HTML file.
+*   The server returns the HTML file and the browser starts parsing it.
+*   The browser encounters the `<link rel="stylesheet">` tag, and starts a new
+    request for the stylesheet.
+*   The browser receives the stylesheet.
+
+With HTTP/2 push:
+
+*   The browser requests the HTML file.
+*   The server returns the HTML file, and pushes the stylesheet at the same
+    time.
+*   The browser starts parsing the HTML. By the time it encounters the `<link
+rel="stylesheet">`, the stylesheet is already in the cache.
+
+In this simplest case, HTTP/2 server push eliminates a single HTTP
+request-response.
+
+With HTTP/1, developers bundle resources together to reduce the number of HTTP
+requests required to render a page. However, bundling can reduce the efficiency
+of the browser's cache. if resources for each page are combined into a single
+bundle, each page gets its own bundle, and the browser can't identify shared
+resources.
+
+The combination of HTTP/2 and HTTP/2 server push can provide the _benefits_ of
+bundling (reduced latency) without needing to bundle resources. Keeping
+resources separate means they can be cached efficiently and be shared between
+pages.
+
+## Conclusion
+
+Loading the code for routes more granularly and allowing browsers to schedule
+work better has the potential to be a holy grail for reaching interactivity in
```

Comment:
I've moved away from this phrasing (you were absolutely correct) to just say that it greatly helps reach interactivity sooner.